app/controllers/entities_controller.rb

```diff
@@ -0,0 +1,20 @@
+class EntitiesController < ApplicationController
+  def new
+    @entity = Entity.new
+  end
+
+  def create
```

Comment:
- [ ] [OPTIONAL] I created a transaction on behalf of another user, we can fix this because it's against the project requirements.